docs/running-a-node/ledger-management.md

```diff
@@ -0,0 +1,141 @@
+# Ledger Management
+
+The node automatically manages the full Nano ledger in the `data.ldb` file which can be found in the data folder at these locations:
+
+--8<-- "folder-locations.md"
+
+This file will grow in size as the ledger does. As of April 2020 there are over 49 million blocks in the ledger which requires at least 26GB of free space. See [hardware recommendations](/running-a-node/node-setup/#hardware-recommendations) for more preferred node specs.
+
+!!! warning "RocksDB uses many files"
+	The above details are for the default LMDB database setup. If using RocksDB, please note that it uses potentially 100s of SST files to manage the ledger so details should followed from the [RocksDB Ledger Backend](#rocksDB-ledger-backend) section below.
+
+## Bootstrapping
+
+When starting a new node the ledger must be downloaded and kept updated in order to participate on the network properly. This is done automatically via bootstrapping - the node downloads and verifies blocks from other nodes across the network. This process can take hours to days to complete depending on network conditions and [hardware specifications](/running-a-node/node-setup/#hardware-recommendations).
+
+### Tuning options
+
+**TODO**: Consider adding notes about tuning here and link to related config entries that need to be updated on configuration page: `bootstrap_connections_max` and `bootstrap_initiator_threads`? Maybe: "Depending on machine and networking resources, the bootstrap performance can be improved by updating X and X configuration values. The additional resource usage these options cause should be considered, especially if left during normal operation (after initial bootstrap is complete)."
```

Comment:
I think this is good to have here, and specifically those two values you mentioned. My advice would be to not go beyond 64 `bootstrap_connections_max` and 4 `bootstrap_initiator_threads`, even for higher end nodes. That should be already plenty fast!

We also have the `--fast_bootstrap` CLI but think we should not mention here, due to causing unconfirmed blocks to not be deleted and increasing the ledger size.

Reply:
Done